markdown/blog/2024-marketing-report.md

```diff
@@ -0,0 +1,129 @@
+
+
+## Closing the Online Store
+
+One significant change is the closure of our online store. After reviewing our budget allocations, it was clear that the store was not generating sufficient activity to justify its expenses. This decision allows us to redirect funds towards initiatives that better serve the community and our core goals. We’re considering alternative ways to share AsyncAPI merchandise in the future that are more cost-effective and in line with our community needs.
+
+As of June 2024, the items in the store were 2383. The goal was to disburse all of the items by December 2024.
+
+![Merchandise in store as at June 2024](/img/posts/marketing-images/merchandise.webp)
+
+ Below is the breakdown of the item distribution and what we have left in the store.
+
+- **Paris Conference:** 1518 items sent to Paris as part of conference giveaways, speakers package, and the rest to be given to the ambassadors to help them promote AsyncAPI in their communities.
+
+- **London Conference:** 335 items shared with attendees and speakers
+
+- **TSC Members:** 40 items packed for the TSC members to acknowledge their contributions to the community.
+
+- **Mentors and Mentees:** 30 items distributed to mentors and mentees as tokens of appreciation for their participation in the mentorship program
+Ambassadors: 2 packages were sent to two new ambassadors, each containing 11 items. 
+
+- **Community Events and Giveaways:** 25 items for various online and offline events throughout the year.
+
+So far, $8,264.37 has been spent on maintaining the store and logistics. As part of the benefit of closing the store, merchandise will be made locally at the event location, and others will be made as per request. [Here](https://docs.google.com/spreadsheets/d/1K7xo-ihlHcVoSYRg1U6SsvQDreEF1S6z/r/edit/edit?gid=1275706868#gid=1275706868) is the link to the distribution board.
+
+![Monthly Spend on The Store](/img/posts/marketing-images/store-report.webp)
+
+Check out our spend so far [here](https://opencollective.com/asyncapi/expenses?type=INVOICE&period=2023-12-31T23%3A00%3A00.000Z%E2%86%92all&searchTerm=Pinnacle+Branding&offset=0&limit=10)
+
+
+## Marketing Strategy Document and Student Ambassador Program
+
+To guide our efforts, we’ve developed a comprehensive [Strategy Document](https://github.com/asyncapi/community/pull/1358) that outlines our objectives, priorities, and target initiatives. This document serves as a roadmap for our upcoming campaigns, events, and community engagement efforts.
+
+Additionally, we’re excited to introduce the [Student Ambassador Program](https://github.com/asyncapi/community/pull/1333), which aims to nurture future AsyncAPI advocates and contributors at the student level. The program will offer students guidance, resources, and community connections to help them grow within the AsyncAPI ecosystem. We believe this will strengthen our community and ensure that AsyncAPI remains a go-to resource for emerging talent.
```

Comment:
I think it's best to specify that we are planning to introduce the student ambassador program, as it is something in the pipeline.